File: Niveau 1/55 - Le juste prix/program.py
nbMarchands = int(input())
position = 0
tempPosition = 0
prix = 1000000
for loop in range(nbMarchands):
   tempPrix = int(input())
   tempPosition = tempPosition + 1
   if tempPrix <= prix:
      prix = tempPrix
      position = tempPosition
print(position)

File: Niveau 1/55 - Le juste prix/program.c
#include <stdio.h>

int main()
{
   int nbMarchands;
   int positionMax = 0;
   scanf("%d", &nbMarchands);
   
   int position = 1;
   int prixMin = 1000 * 1000;
   for (int i = 1; i <= nbMarchands; i++)
   {
      int prix;
      scanf("%d", &prix);
      
      if (prix <= prixMin)
      {
         prixMin = prix;
         positionMax = position;
      }
      position++;
   }
   printf("%d", positionMax);
 
   return 0;
}
